cmake/cvars.cmake

```diff
@@ -0,0 +1,50 @@
+set(CVAR_VSYNC_ENABLED "gVsyncEnabled" CACHE STRING "")
+set(CVAR_Z_FIGHTING_MODE "gZFightingMode" CACHE STRING "")
+set(CVAR_NEW_FILE_DROPPED "gNewFileDropped" CACHE STRING "")
+set(CVAR_DROPPED_FILE "gDroppedFile" CACHE STRING "")
+set(CVAR_INTERNAL_RESOLUTION "gInternalResolution" CACHE STRING "")
+set(CVAR_MSAA_VALUE "gMSAAValue" CACHE STRING "")
+set(CVAR_SDL_WINDOWED_FULLSCREEN "gSdlWindowedFullscreen" CACHE STRING "")
+set(CVAR_TEXTURE_FILTER "gTextureFilter" CACHE STRING "")
+set(CVAR_CONTROL_NAV "gControlNav" CACHE STRING "")
+set(CVAR_CONSOLE_ENABLED "gConsoleEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_CONFIGURATION_ENABLED "gControllerConfigurationEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_DISCONNECTED_ENABLED "gControllerDisconnectedWindowEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_REORDERING_ENABLED "gControllerReorderingWindowEnabled" CACHE STRING "")
+set(CVAR_GFX_DEBUGGER_ENABLED "gGfxDebuggerEnabled" CACHE STRING "")
+set(CVAR_STATS_ENABLED "gStatsEnabled" CACHE STRING "")
+set(CVAR_ENABLE_MULTI_VIEWPORTS "gEnableMultiViewports" CACHE STRING "")
+set(CVAR_LOW_RES_MODE "gLowResMode" CACHE STRING "")
+set(CVAR_SIMULATED_INPUT_LAG "gSimulatedInputLag" CACHE STRING "")
+set(CVAR_ALT_ASSETS "gAltAssets" CACHE STRING "")
+set(CVAR_SINK_ENABLED "gSinkEnabled" CACHE STRING "")
```

Comment:
i was hoping to see if i could think of a more descriptive name for this but i have no idea what it does, it seems to only be used here

https://github.com/Kenix3/libultraship/blob/a245aaf975f1f3c85ecd63ac3a9d3e8b44949eb8/src/window/gui/ConsoleWindow.cpp#L322-L339

Reply:
so it seems this functionality was removed/broken at some point, i think for now the best thing to do is just not make a macro for `gSinkEnabled`, and i'll make an issue to figure out what's going on with the ability to send logs to the in game console